types.bal

```diff
@@ -90,7 +90,7 @@ type Pledge record {
     string donorID;
     decimal amount;
     string status;
-    Donor? donor;
+    Donor donor;
```

Comment:
Will the Pledge apis break if there is no display name for the user ?
We currently haven't set display names for all.

Reply:
Yes, if the `displayName` is not available for a donor then it will break at SCHIM client level. Hence, we will have to update the `displayName` for all donors or else we can mark `displayName` as a optional field at `Donor` record [1]

[1] - [https://github.com/LSFLK/MedicinesforLK-AdminAPI/blob/c2aac1f33b852231ed5f76c69e3a79e1333bd3f6/types.bal#L20](https://github.com/LSFLK/MedicinesforLK-AdminAPI/blob/c2aac1f33b852231ed5f76c69e3a79e1333bd3f6/types.bal#L20)

Reply:
I don't think we can break the whole pledge api due to one user not having a display name. We ether need 

1) donor to be optional
Or
2) 
> mark displayName as an optional field at Donor record 

But 2 require front end change.

Reply:
> I don't think we can break the whole pledge api due to one user not having a display name.  

Agreed

Let's make donor as an optional field in `Pledge` record.

Reply:
Fixed with [e3a01810d91e1d79ffb64db3ac8d0968571df728](https://github.com/LSFLK/MedicinesforLK-AdminAPI/pull/55/commits/e3a01810d91e1d79ffb64db3ac8d0968571df728)